src/dbSta/src/dbReadVerilog.cc

```diff
@@ -270,8 +317,18 @@ void Verilog2db::makeDbModule(Instance* inst, dbModule* parent)
     module = block_->getTopModule();
   } else {
     module = makeUniqueDbModule(network_->name(cell));
+    inst_module_vec.emplace_back(
+        std::pair<const Instance*, dbModule*>(inst, parent));
+
+    std::string module_inst_name = network_->name(inst);
+    size_t last_idx = module_inst_name.find_last_of('/');
+    if (last_idx != string::npos) {
+      module_inst_name = module_inst_name.substr(last_idx + 1);
+    }
```

Comment:
I tried experimenting with read_verilog2.tcl but I never saw this do anything (there are no /).  I added an extra level of hierarchy but still nothing.  When does this occur?

Reply:
Names are locally scoped. So if there is a module instance U1 inside another called U0 then the module instance is still U1. The hierarchical names are constructed elsewhere. 

To help you experiment with this a test has been added called : readdb_hier.tcl.

This prints out some names for you to look at. But again, note that we are using locally scoped names. The full names are resolved in the dbNetwork.cc.